lib/core/data.coffee

```diff
@@ -0,0 +1,39 @@
+KDEventEmitter = require './eventemitter'
+
+module.exports = class KDData extends KDEventEmitter
```

Comment:
I'd love to have more docs, and possibly tests covering this class, because I don't even understand the need of a `Proxy` object here.